sros2/sros2/api/_artifact_generation.py

```diff
@@ -24,18 +24,19 @@
 def generate_artifacts(
         keystore_path: pathlib.Path = None,
         identity_names: List[str] = [],
-        policy_files: List[pathlib.Path] = []) -> None:
+        policy_files: List[pathlib.Path] = [],
+        domain_id: int = 0) -> None:
```

Comment:
By setting the default value to 0 here, doesn't this bypass the fallback to the environment variable ? It should be None here too isn't it?


Adding a corresponding test to the test suite would be valuable to validate the behavior

Reply:
Good catch, I'll get this fixed and add test cases.